app/services/xccdf_report_migration.rb

```diff
@@ -1,141 +0,0 @@
-# frozen_string_literal: true
-
-# Migrates ref IDs from reports to comply with the expected standard.
-# Profile ref IDs should look like "xccdf_org.ssgproject.content_profile_*"
-# Rule ref IDs should look like "xccdf_org.ssgproject.content_rule_*"
-#
-# Disabling ClassLength here as the migration is likely going to be removed
-# at some point since we don't accept these reports anymore and old data was
-# already migrated
-# rubocop:disable Metrics/ClassLength
-class XCCDFReportMigration
```

Comment:
I guess we'll always have it in our git history if it's ever needed again (not that it should be needed ever again).